tests/src/performance/perflab/LowLevelPerf.cs

```diff
@@ -0,0 +1,765 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Xunit.Performance;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Runtime.CompilerServices;
+
+public class LowLevelPerf
+{
+    [Benchmark]
+    public static void EmptyStaticFunction()
+    {
+        foreach (var iteration in Benchmark.Iterations)
+            using (iteration.StartMeasurement())
+                Class.EmptyStaticFunction();
```

Comment:
Is the overhead of StartMeasurement and Dispose and whatnot low enough that calling EmptyStaticFunction just once is actually meaningful?  I'd have expected here and in some of the cases below that we'd want to call EmptyStaticFunction a bunch of times within the same iteration.